### PR TITLE
Fix duplicate keys in medical, Close #3372

### LINKED ISSUE #3372: Duplicated stringtable entries on medical
```
 0:55:56 Item STR_ACE_Medical_Atropine_Display listed twice
0:55:56 Unexpected stringtable format inside <Text ID="STR_ACE_Medical_Activity_cpr"><Key>
0:55:56 Item STR_ACE_Medical_Activity_CPR listed twice
```

### PR DESCRIPTION
**Pull request by bux578**
### When merged this pull request will:

1. Fix the duplicate keys in medical
2. comment out the other translation because the wording has changed

### CODE REVIEW

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -965,16 +962,16 @@
             <English>A drug used to counter the effects of Epinephrine</English>
         </Key>
         <Key ID="STR_ACE_Medical_Atropine_Display">
-            <English>Atropine autoinjector</English>
```
Comment (Glowbal) [inline addons/medical/stringtable.xml]: This should not be deleted

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -955,9 +955,6 @@
             <Portuguese>Um analgésico usado para combater dores moderadas e fortes.</Portuguese>
             <Czech>Analgetikum slouží k tlumení středně těžkých a těžkých bolestí</Czech>
         </Key>
-        <Key ID="STR_ACE_Medical_Atropine_Display">
```
Comment (Glowbal) [inline addons/medical/stringtable.xml]: This key has to renamed to `STR_ACE_Medical_Atropine_Display`

Comment (Glowbal) [inline addons/medical/stringtable.xml]: And of course I wrote atropine instead of Adenosine here again..

Comment (bux578) [inline addons/medical/stringtable.xml]: You mean `STR_ACE_Medical_Adenosine_Display`

**Pull request closed without merging**